internal/app/singularity/cache_list_linux.go

```diff
@@ -31,82 +34,98 @@ func findSize(size int64) (string, error) {
 	return "", fmt.Errorf("failed to detect file size")
 }
 
-func listLibraryCache() error {
-	// loop through library cache
+// listLibraryCache will loop throught and list all library cache (~/.singularity/cache/library).
+// will return the amount of library containers, the total space thoughts containers are using,
+// and an error if one occures.
```

Comment:
and an error if one **occurs**

Reply:
"the number of containers in the library"?

"the total space these containers"?